bench/test_serialize.py

```diff
@@ -0,0 +1,52 @@
+from bench.utils.factories import FRAME_FACTORIES
+from bench.utils import get_int
+
+
+NUMBER_OF_FRAMES = 100
+
+
+def _serialize_function(frames):
+    frame = frames[get_int(0, NUMBER_OF_FRAMES - 1)]
+    frame.serialize()
+
+
+def _create_frames(frame_type):
+    return [FRAME_FACTORIES[frame_type]() for _ in range(100)]
+
+
+class TestSerializeBenchmarks:
+    def test_serialize_data_frame(self, benchmark):
+        benchmark(_serialize_function, _create_frames("data"))
```

Comment:
Given that pytest is already in use elsewhere in the codebase, I wonder if it would be worth condensing this using `@pytest.mark.parametrize`, something like:

```python
@pytest.mark.parametrize('frame_name', [
    'data',
    'priority',
    ...,
    'rst_stream'
])
def test_serialize_frame(frame_name, benchmark):
    benchmark(_serialize_function, _create_frames(frame_name))
```